apps/web/modules/ee/audit-logs/lib/service.ts

```
@@ -0,0 +1,44 @@
+      }
+
+      this.validateEvent(event);
+
+      auditLogger.info(event);
+    } catch (error) {
+      // Log error to application logger but don't throw
+      // This ensures audit logging failures don't break the application
+      logger.error(error, "Failed to log audit event");
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)